infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/GitUserProfileProvisioner.java

```diff
@@ -82,7 +82,7 @@ public void provision(KubernetesEnvironment k8sEnv, RuntimeIdentity identity)
   }
 
   private Map<String, String> getMapFromJsonObject(String json) {
-    Type stringMapType = new TypeToken<Map<String, String>>() {}.getType();
+    Type stringMapType = new TypeToken<Map<String, Object>>() {}.getType();
```

Comment:
Should this variable name be updated as well?

Reply:
Updated